### DISCUSSION TITLE
How to Map LDAP Groups?

### DISCUSSION THREAD

**Discussion by Marco111111111111** [requarks/wiki#6181] [Help / Questions]
Hello,
does anyone got LDAP Group Mapping to work?
Ive tested some things, but nothing worked.
This is my config:
![Wiki](https://user-images.githubusercontent.com/125453471/220024014-f756c5e2-2d23-4b94-8ba2-2ec4ed6695b9.png)

Any suggestions?

**Comment by q-wertz** [comment]
See https://github.com/requarks/wiki/discussions/6139 .
Looks like there is a bug with existing authentication strategies (see NGPixels answer). You can create a new authentication strategy and there the settings appear (at least in my case).

**Comment by icsinfo** [comment]
This one will be hard to answer without seeing your AD setup, but the groups must also exist in the wiki for it to map to them. It will not create the groups in the wiki for you.
